src/api/track.ts

```diff
@@ -66,6 +66,13 @@ class TrackAPI {
       tripOptions.version = '2';
     }
 
+    var reqHeaders: { 'X-Radar-Product'?:string } = {}
```

Comment:
Lets define the type at the top of this file with the other types:
```ts
interface TrackRequestHeaders {
  'X-Radar-Body-Is-Token'?: boolean;
  'X-Radar-Product'?: string;
}
```

Then we can create the object like:
```ts
const headers: TrackRequestHeaders = {};

if (fraud) {
  headers['X-Radar-Body-Is-Token'] = true;
}

if (product) {
  headers['X-Radar-Product'] = product;
}
```